### PR TITLE
ICONS: Change build config

### LINKED ISSUE #203: Support SVG attributes in JSX
During build process of icons the svg attributes should be transformed to JSX compliant ones. 
See the list of JSX compliant attributes at the end of this page https://reactjs.org/docs/dom-elements.html

suggestion: use https://github.com/smooth-code/svgr in build process

Below is an example of wrongly generated Loading icon component (the second circle tags has `fill-opacity` attribute, which should be `fillOpacity` instead)

```javascript

/* eslint-disable */
import * as React from "react";
import OrbitIcon from "../Icon";

export default function Loading(props) {
  const { color, size, customColor } = props;
  return (
    <OrbitIcon viewBox="0 0 24 24" size={size} color={color} customColor={customColor}>
      <circle cx="4.308" cy="12" r="2.308">
        <animate
          attributeName="fill-opacity"
          begin="0s"
          dur="1s"
          values="1;.2;1"
          calcMode="linear"
          repeatCount="indefinite"
        />
      </circle>
      <circle cx="4.308" cy="4.308" r="2.308" fill-opacity=".5">
        <animate
          attributeName="fill-opacity"
          begin="100ms"
          dur="1s"
          values="1;.2;1"
          calcMode="linear"
          repeatCount="indefinite"
        />
      </circle>
      <circle cx="12" cy="4.308" r="2.308">
        <animate
          attributeName="fill-opacity"
          begin="300ms"
          dur="1s"
          values="1;.2;1"
          calcMode="linear"
          repeatCount="indefinite"
        />
      </circle>
      <circle cx="12" cy="12" r="2.308">
        <animate
          attributeName="fill-opacity"
          begin="600ms"
          dur="1s"
          values="1;.2;1"
          calcMode="linear"
          repeatCount="indefinite"
        />
      </circle>
      <circle cx="19.692" cy="4.308" r="2.308">
        <animate
          attributeName="fill-opacity"
          begin="800ms"
          dur="1s"
          values="1;.2;1"
          calcMode="linear"
          repeatCount="indefinite"
        />
      </circle>
      <circle cx="19.692" cy="12" r="2.308">
        <animate
          attributeName="fill-opacity"
          begin="400ms"
          dur="1s"
          values="1;.2;1"
          calcMode="linear"
          repeatCount="indefinite"
        />
      </circle>
      <circle cx="4.308" cy="19.692" r="2.308">
        <animate
          attributeName="fill-opacity"
          begin="700ms"
          dur="1s"
          values="1;.2;1"
          calcMode="linear"
          repeatCount="indefinite"
        />
      </circle>
      <circle cx="12" cy="19.692" r="2.308">
        <animate
          attributeName="fill-opacity"
          begin="500ms"
          dur="1s"
          values="1;.2;1"
          calcMode="linear"
          repeatCount="indefinite"
        />
      </circle>
      <circle cx="19.692" cy="19.692" r="2.308">
        <animate
          attributeName="fill-opacity"
          begin="200ms"
          dur="1s"
          values="1;.2;1"
          calcMode="linear"
          repeatCount="indefinite"
        />
      </circle>
    </OrbitIcon>
  );
}

```

### PR DESCRIPTION
**Pull request by wEprous**
Changed icons build config.

- Config is now using SVGR which generates icons components with proper JSX attributes.
- Fixed typo `terciary` => `tertiary`.
- Added flow declaration for every icon.

closes #203

### CODE REVIEW

**Comment by Fallion** [comment]
Svgr no? Please fix description.

**Comment by wEprous** [comment]
Yep, fixed, sry @Fallion

**Comment by tomashapl** [comment]
@wEprous can you please update this PR ?

File: src/Icon/consts.js [hunk 1]
```diff
@@ -11,5 +11,5 @@ export const ICON_COLORS = {
   attention: defaultTokens.orbit.colorIconAttention,
   primary: defaultTokens.orbit.colorIconPrimary,
   secondary: defaultTokens.orbit.colorIconSecondary,
-  terciary: defaultTokens.orbit.colorIconTerciary,
+  tertiary: defaultTokens.orbit.colorIconTerciary,
```
Comment (tomashapl) [inline src/Icon/consts.js]: we should fix also tokens naming

Comment (wEprous) [reply src/Icon/consts.js]: Cannot right know, I have it in my TODOs for new tokens version.

**Pull request merged**